src/libs/freetype/lv_freetype_outline.c

```diff
@@ -354,6 +360,40 @@ static lv_freetype_outline_t outline_create(
         .delta = 0
     };
 
+#if LV_USE_NEMA_GFX
+#if LV_USE_NEMA_VG
+    /*Calculate Total Segmenets Before decompose */
+    int tag_size = face->glyph->outline.n_points;
```

Comment:
The outline path length information can be stored in the `lv_freetype_outline_event_param_t` structure, so that the length information can be read in `LV_EVENT_CREATE`, avoiding coupling of NEMA's private data structure in the freetype code.